crates/pathfinder/src/cairo/ext_py/ser.rs

```diff
@@ -40,13 +40,16 @@ pub(crate) enum UsedChain {
     Mainnet,
     #[serde(rename = "GOERLI")]
     Goerli,
+    #[serde(rename = "INTEGRATION")]
+    Integration,
```

Comment:
I think this requires something equivalent on the python side; but I wasn't entirely sure of the implications or if we even care?

Reply:
Yeah please don't add that. I don't think there is one.

Reply:
Confirmed, the related enum only has two members. It's about the where fees are deposited only, sadly. Fee charging should not happen during call or estimate_fee.

Reply:
Okay so I should map `Chain::Integration` to `Goerli` here instead?